### PR TITLE
Feature/ci

### PR DESCRIPTION
**Pull request by trasher**

### CODE REVIEW

File: phpstan.neon [hunk 1]
```diff
@@ -0,0 +1,17 @@
+parameters:
+    parallel:
+        maximumNumberOfProcesses: 2
+    level: 1
+    bootstrapFiles:
+        - ../../inc/based_config.php
+        - setup.php
```
Comment (cedric-anne) [inline phpstan.neon:7]: Should be moved in `paths` entry.

**Comment by cedric-anne** [comment]
> inc/computergroupdynamic.class.php

Maybe replace `get_class($item) == PluginDatabaseinventoryComputerGroup::getType()` by `$item instanceof PluginDatabaseinventoryComputerGroup`.

**Comment by trasher** [comment]
Remaining issues are caused from a core phpdoc error

**Comment by trasher** [comment]
Fixed by https://github.com/glpi-project/glpi/pull/15953

**Pull request merged**